tests/unit/utils/test_wsgi.py

```diff
@@ -130,13 +138,14 @@ def test_accepts_x_forwarded_headers(self, remote_addr_salted):
 
         resp = wsgi.ProxyFixer(app, token=None, ip_salt="pepa")(environ, start_response)
 
+        expected_remote_addr_hashed = calculate_hashed_value("1.2.3.4", "pepa")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** Can you help me understand why we're back to using a 1.2.3.4 here?
In my head, all the `REMOTE_ADDR` entries above could be replaced by the `remove_addr` fixture, and the `REMOTE_ADDR_HASHED` below could be replaced with the `remote_addr_hashed` fixture, and the `calculate_hashed_value()` function could go away.‏